networks/_infra/aws-ec2/main.tf

```diff
@@ -44,7 +44,7 @@ data "aws_ami" "this" {
 }
 
 data "http" "myip" {
-  url = "https://bot.whatismyipaddress.com"
+  url = "https://ifconfig.co/"
```

Comment:
I found `https://checkip.amazonaws.com` is more reliable. 
Need to `chomp` it as the output contains `\n` at the end.